Send survey utility & default num_tries

rebecca-caktus opened the pull request:
- Makes the default of `Session.num_tries` 0. 
- Adds a generic utility to send a survey to a batch of connections.

Commit:
Create default for num_tries

Commit:
Add start_survey util

Commit:
Create TranscriptMessage model

Commit:
Hook up multitenancy for transcript message

Commit:
Refactor common code in multitenancy signals

Commit:
Record transcript messages

Commit:
Create transcript message in start_survey()

Commit:
Add tests for TranscriptMessage

Commit:
Add answer color to answer list view

ghost commented:
Some other utilities for theirc/rescuesms#124 got tacked onto this pull as well.

Commit:
Don't inherit from TenantEnabled

Because it doesn't always make sense to be required to use the
by_tenant or all_tenants manager. Requires no migrations.

Commit:
Factor out generic create_tenant_link utility to utils file

So that it can be more easily used elsewhere.

Commit:
Fix Flake8
